boa_gc/src/lib.rs

```diff
@@ -234,6 +238,32 @@ impl Allocator {
         })
     }
 
+    fn alloc_weak_map<K: Trace, V: Trace>() -> WeakMap<K, V> {
+        let _timer = Profiler::global().start_event("New WeakGc", "BoaAlloc");
```

Comment:
The profiler title should probably be "New WeakMap" here, right?

Reply:
Good catch, fixed